src/api/spec/models/project_spec.rb

```diff
@@ -501,6 +501,10 @@
       create(:relationship, project: project, user: user)
     end
 
+    before do
+      allow(project).to receive(:check_write_access!)
```

Comment:
What about trying to set ``User.current = user`` first? Then we could even extend the test and set another user and check that this user is not able to delete?

Reply:
What would I set User.current to in the #remove_all_groups test?

Reply:
You could create a groups_user object and use this one
```
let(:groups_user) { create(:groups_user, group: group) }
let(:user) { groups_user.user }
```

Does that makes sense?

Reply:
Done.